SPECIFICATIONS.md

```diff
@@ -720,7 +720,27 @@ The symbol table is created from a default table containing, in order:
 - operation
 - right
 - time
-- revocation_id
+- rule
```

Comment:
In the issue it was `role` (for rbac). `rule` might be interesting as well though